components/omega/doc/design/SubmesoscaleEddies.md

```diff
@@ -0,0 +1,345 @@
+
+Overall, the Uchida et al. study demonstrates that the physically-based constraints introduced by Bodner et al. lead to a more realistic and stable climate simulation.
+
+## 2. Requirements
+
+### 2.1 Requirement: Buoyancy calculation must be modular
+
+To allow for easier addition of a future mesoscale eddy closure ([e.g. Mak et al 2018](https://journals.ametsoc.org/view/journals/phoc/48/10/jpo-d-18-0017.1.xml) and/or [Korn et al 2018](https://www.sciencedirect.com/science/article/pii/S1463500318301859)) the buoyancy calculation and horizontal gradient that is required for the submesoscale eddy parameterization ([](#submeso-definition)) should be usable for mesoscale eddy closures.  For a non-boussinesq model, the buoyancy is defined as
+
+$$
+b \equiv g \frac{\rho - \rho_0}{\rho}
+$$
+
+where $\rho_0$ is a constant reference density.  The vertical mixed layer average, where $H$ is the mixed layer depth, in pseudo-height coordinates is given by
+
+$$
+b_{ml} = \frac{1}{H} \int_{-H}^0 g\frac{\rho - \rho_0}{\rho} d \tilde{z}
+$$
+
+This can be rewritten as
+
+$$
+b_{ml} = \frac{1}{H}\int_{-H}^0 g d\tilde{z} - \frac{1}{H}\int_{-H}^0 g \frac{\rho_0}{\rho} d \tilde{z}
+$$
+
+since $g$ is constant, we can write
+
+$$
+b_{ml} = g  - \frac{1}{H}\int_{-H}^0 g \frac{\rho_0}{\rho} d \tilde{z} = g - \frac{1}{H}\int_{-H}^0 g \rho_0 \alpha
+$$
+
+where $\alpha$ is the specific volume.
+
+### 2.2 Requirement: Buoyancy calculation must account for tilted pressure coordinates
+
+As in MPAS-Ocean, the horizontal buoyancy gradient calculation must supported tilted coordinates.  In the psuedo height coordinate for Omega, the horizontal gradient of buoyancy for a tilting surface ($\tilde{z}$) is
+
+$$
+\nabla b = \nabla_h b + \frac{\partial b}{\partial \tilde{z}} \nabla_h \tilde{z}
+$$
+
+where $\nabla_h$ is a gradient along a fixed model layer in Omega.  The vertical derivative of buoyancy will utilize the Brunt Vaisala Frequency variable from the vertical mixing calculation.
+
+### 2.3 Requirement: The parameterization must be configurable in the traditiona FK11 form and the B23 update
+
+The parameterization of submesoscale eddies must accept fixed parameters from the input Yaml file for things like the efficiency ($C_e$), the frontal width ($L_f$) and the inverse timescale ($\tau$) to be consistent with FK.  Via a configuration flag the parameterization should use the B23 form with dynamic predictions of these key variables.  The B23 update requires an estimate of boundary layer depth and thus will be implemented after the K-Profile Parameterization of vertical mixing is implemented in Omega.
+
+## 3. Algorithmic Formulation
+
+In Omega two forms of the submesoscale eddy parameterization will be included: FK11 and B23.
+
+### 3.1 The original FK11 formulation
```

Comment:
This section is a bit confusing when it comes to $\tau$. Is the $\tau$ in Equation 1 the same as the $\tau$ here or the same as $\tau_{min}$?

Reply:
I've tried to address this by only having a \tau_MLI and wind stress is \tau.  Hopefully this helps